tests/UnifiedSpecTests/CollectionData.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace MongoDB\Tests\UnifiedSpecTests;
+
+use ArrayIterator;
+use IteratorIterator;
+use MongoDB\Client;
+use MongoDB\Driver\ReadConcern;
+use MongoDB\Driver\ReadPreference;
+use MongoDB\Driver\WriteConcern;
+use MongoDB\Tests\UnifiedSpecTests\Constraint\Matches;
+use MultipleIterator;
+use stdClass;
+use function sprintf;
+
+class CollectionData
+{
+    /** @var string */
+    private $collectionName;
+
+    /** @var string */
+    private $databaseName;
+
+    /** @var array */
+    private $documents;
+
+    public function __construct(stdClass $o)
+    {
+        assertInternalType('string', $o->collectionName);
+        $this->collectionName = $o->collectionName;
+
+        assertInternalType('string', $o->databaseName);
+        $this->databaseName = $o->databaseName;
+
+        assertInternalType('array', $o->documents);
+        assertContainsOnly('object', $o->documents);
+        $this->documents = $o->documents;
+    }
+
+    /**
+     * Prepare collection state for "initialData".
+     *
+     * @param Client $client
+     */
+    public function prepare(Client $client)
+    {
+        $database = $client->selectDatabase(
+            $this->databaseName,
+            ['writeConcern' => new WriteConcern(WriteConcern::MAJORITY)]
+        );
+
+        $database->dropCollection($this->collectionName);
```

Comment:
That commit isn't associated with any branch. Do you have more context for that?

Collections are only dropped if they appear in `initialData`, so this seems like something I may want to specify in the spec itself (e.g. "SHOULD NOT use admin collections in `initialData`), since nothing is currently stopping a test from doing so.

Reply:
GitHub seems to mess up the link to the commit in the PR. The commit was part of #780, but viewing the commit standalone shows it as not being part of a branch due to squashed commits. Not sure I like that part about commit squashing...

Reply:
For more context, the problem was crud/db-aggregate.json, which uses the admin database to run an aggregate with the `$listLocalSessions` pipeline stage. I'm not aware of a limitation that would require running this pipeline stage against the admin database, so adding a limitation in the spec should also suffice.

Reply:
Ok, that shouldn't be a problem then. Merely creating an enttiy for the admin database won't have the unified test runner drop it, and a test for `$listLocalSessions` wouldn't need to use `initialData` on an admin collection (or any collection for that matter).

What you're referring to was likely the same thing I ran into while writing tests for Atlas Data Lake, as the test runner attempted to drop the collection under test by default (which unmapped the data in `mongohoused`).

Reply:
Ah, yes, that's the problem then. In that case, nothing to do here 👍